showcase/tests/integration/components/hds/form/key-value-inputs/field-test.js

```diff
@@ -22,40 +91,60 @@ module(
         .hasClass('hds-form-key-value-inputs__field');
     });
 
-    test('it should render the subcomponents with the correct CSS classes', async function (assert) {
-      await render(hbs`<Hds::Form::KeyValueInputs::Field id="test-form-key-value-field" @rowIndex={{0}} as |F|>
-      <F.Label>Label</F.Label>
-      <F.TextInput @value="Value" />
-      <F.HelperText>Helper text</F.HelperText>
-      <F.Error>Error text</F.Error>
-    </Hds::Form::KeyValueInputs::Field>`);
+    // LABEL HIDDEN TEXT
+
+    test('it should render the label with a screen-reader-only text', async function (assert) {
```

Comment:
```suggestion
    test('it should render the label with screen-reader-only text based on the provided `@rowIndex` argument', async function (assert) {
```
[Nit] indicate that this test also involves testing the `@rowIndex` arg